packages/enzyme-matchers/src/assertions/toMatchElement.js

```diff
@@ -7,13 +7,14 @@
  */
 
 import { shallow, mount } from 'enzyme';
-import type { EnzymeObject, Matcher } from '../types';
+import type { EnzymeObject, Matcher, ToMatchElementOptions } from '../types';
 import isShallowWrapper from '../utils/isShallowWrapper';
 import single from '../utils/single';
 
 function toMatchElement(
   actualEnzymeWrapper: EnzymeObject,
-  reactInstance: Object
+  reactInstance: Object,
+  options: ToMatchElementOptions = { ignoreProps: true }
```

Comment:
It seems that setting this property might become a breaking change, right? I think that's fine. I just want to make sure I release it properly.

Reply:
It's a new 3rd param that always gets a default value if none is passed in. I guess if someone is currently calling this method with an unused third param and it's not of type object, then it could break. But that's just bad code IMO, not a breaking change.

Reply:
I can see that it *could* possibly break some cases but seems very unlikely.

Reply:
I forget what the previous code did. Before this change, was the behavior of `toMatchElement` that it ignored props?

Reply:
Ah - you are correct, it did not ignore props before, so this will be breaking. Thank you for catching this.

Reply:
Cool. No worries. Just wanted to make sure I understood. So this is fine. I feel like it's the right default. A new major is fine.